### DISCUSSION TITLE
Idea: Add Cost-Per-Drive Calculation for Electricity and Gas (Like Tessie)

### DISCUSSION THREAD

**Discussion by giterdone509** [teslamate-org/teslamate#4398] [Ideas]
I would love a feature similar to what Tessie offers, with cost comparisons for electric vs. gas, a drive stats dashboard, and charge-to-fuel cost insights to help users better understand their costs and savings. 

Drives 

- Cost per drive in both electric and gas. The cost per litre or gallon is inputted by the user based on the average cost of gas in their area for each month. 
- Drive stats and statistics dashboard to aggregate data  

Tessie screenshot 
![image](https://github.com/user-attachments/assets/04d5b683-75e0-45a8-91d5-64077833682a)

Charges 

- In addition to the cost per charge that already exists, add a feature to display what that charge would have costed in fuel.  

Tessie screenshot 
![image](https://github.com/user-attachments/assets/1b02ee02-22f8-42bd-9160-931a6ba01d9b)

**Comment by shifty8k** [comment]
Sounds like a great idea! Would love to see this.
